sandy/libraries.py

```diff
@@ -14661,3 +14661,115 @@
     1102731: "decay_3851_110-Ds-273M.dat",
     1112720: "decay_3852_111-Rg-272.dat",
 }
+
+# Thermal Neutron Scattering Data for ENDF-B-VII.1 from the IAEA website
+URL_TSL_ENDFB_71_IAEA = "https://www-nds.iaea.org/public/download-endf/ENDF-B-VII.1/tsl/"
+TSL_FILES_ENDFB_71_IAEA = {
+    1: "tsl_0001_H(H2O).dat",
```

Comment:
Unfortunately the MAT numbers provided at page 673 of the LEAPR section  of the NJOY manual are not meaningful.
I think we cannot  anymore use integers (like the ZAM number) to identify the compounds. We need real names with strings.

I propose using those reported in the same page of the NJOY manual.
Example: `water`, `BeO`, `Be_in_BeO`.

Probably we need a function that tells the users what keys are available for TSL data, otherwise it would be too complicated to find the right name and match the correct case-sensitive key.

Reply:
In my opinion that only makes it more difficult to write and use the code.I agree with both Enrica and Luca that we need something that relates the `zam` of the method to the keys available for TSL. So my question is the following:

If we create a function that gives you everything: the keys and their meaning, why don't we call them with integers? After all, if such a function exists, the key would be decided by us and the user would know it. 

My opinion is the following: Appear in the docstring the whole list (as Enrica said), and change line 326 so that in the case of tsl it prints the whole dictionary with the raise error and in the other cases, it returns the closest values that are available in the library.